### PR TITLE
Adjust phrasing of features

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Separate feature description and where it is enabled into two sentences, following suggestion by @devoncarew . Adjust names of features to better match brand guidelines.

```
Usage: flutter config [arguments]
-h, --help                               Print this usage information.
    --[no-]analytics                     Enable or disable reporting anonymously tool usage statistics and crash reports.
    --clear-ios-signing-cert             Clear the saved development certificate choice used to sign apps for iOS device
                                         deployment.

    --gradle-dir                         The gradle install directory.
    --android-sdk                        The Android SDK directory.
    --android-studio-dir                 The Android Studio install directory.
    --[no-]enable-web                    Enable or disable Flutter for Web. This setting will take effect on master, dev
                                         channels.

    --[no-]enable-linux-desktop          Enable or disable Flutter for Linux Desktop. This setting will take effect on the
                                         master channel.

    --[no-]enable-macos-desktop          Enable or disable Flutter for macOS Desktop. This setting will take effect on the
                                         master channel.

    --[no-]enable-windows-desktop        Enable or disable Flutter for Windows Desktop. This setting will take effect on the
                                         master channel.

    --[no-]enable-build-plugin-as-aar    Enable or disable Build plugins independently as AARs in app projects. This setting
                                         will take effect on master, dev channels.

    --clear-features                     Remove all configured features and restore them to the default values.
```

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
lgtm

File: packages/flutter_tools/test/general.shard/features_test.dart [hunk 1]
```diff
@@ -78,19 +78,19 @@ void main() {
     }));
 
     test('flutter web help string', () {
-      expect(flutterWebFeature.generateHelpMessage(), 'Enable or disable Flutter Web on master, dev channels.');
+      expect(flutterWebFeature.generateHelpMessage(), 'Enable or disable Flutter for Web. This setting will take effect on master, dev channels.');
```
Comment (devoncarew) [inline packages/flutter_tools/test/general.shard/features_test.dart]: `"Flutter for Web"` or `"Flutter for web"`?

And, perhaps `"This setting will take effect on the master and dev channels.".

**Review by devoncarew — APPROVED** [review]
lgtm w/ some potential additional word-smithing

File: packages/flutter_tools/lib/src/features.dart [hunk 2]
```diff
@@ -73,7 +73,7 @@ const List<Feature> allFeatures = <Feature>[
 
 /// The [Feature] for flutter web.
 const Feature flutterWebFeature = Feature(
-  name: 'Flutter Web',
+  name: 'Flutter for Web',
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/features.dart]: Also here, "Flutter for Web" ==> "Flutter for web"

**Comment by jonahwilliams** [comment]
Updated usage: 

```
Usage: flutter config [arguments]
-h, --help                               Print this usage information.
    --[no-]analytics                     Enable or disable reporting anonymously tool usage statistics and crash reports.
    --clear-ios-signing-cert             Clear the saved development certificate choice used to sign apps for iOS device
                                         deployment.

    --gradle-dir                         The gradle install directory.
    --android-sdk                        The Android SDK directory.
    --android-studio-dir                 The Android Studio install directory.
    --[no-]enable-web                    Enable or disable Flutter for web. This setting will take effect on the master and dev
                                         channels.

    --[no-]enable-linux-desktop          Enable or disable Flutter for Linux desktop. This setting will take effect on the
                                         master channel.

    --[no-]enable-macos-desktop          Enable or disable Flutter for macOS desktop. This setting will take effect on the
                                         master channel.

    --[no-]enable-windows-desktop        Enable or disable Flutter for Windows desktop. This setting will take effect on the
                                         master channel.

    --[no-]enable-build-plugin-as-aar    Enable or disable Build plugins independently as AARs in app projects. This setting
                                         will take effect on the master and dev channels.

    --clear-features                     Remove all configured features and restore them to the default values.

Run "flutter help" to see global options.

```

File: packages/flutter_tools/lib/src/features.dart [hunk 3]
```diff
@@ -88,7 +88,7 @@ const Feature flutterWebFeature = Feature(
 
 /// The [Feature] for macOS desktop.
 const Feature flutterMacOSDesktopFeature = Feature(
-  name: 'Flutter Desktop for macOS',
+  name: 'Flutter for macOS desktop',
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/features.dart]: `"Flutter for macOS desktop"` ==> `"Flutter for desktop on macOS"`? `"Flutter for desktop (macOS)"`?

The changes above would let the name read `Flutter for desktop` (which would parallel the `Flutter for web` name).

lgtm either way

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/features.dart]: Done

**Pull request merged**